node/src/neighborhood/mod.rs

```diff
@@ -2627,6 +2634,31 @@ mod tests {
         assert_eq!(result, None);
     }
 
+    #[test]
+    fn route_query_responds_with_none_when_wildcard_ip_is_requested() {
+        init_test_logging();
+        let test_name = "route_query_responds_with_none_when_wildcard_ip_is_requested";
+        let system = System::new(test_name);
+        let mut subject = make_standard_subject();
+        subject.logger = Logger::new(test_name);
+        let addr: Addr<Neighborhood> = subject.start();
+        let sub: Recipient<RouteQueryMessage> = addr.recipient::<RouteQueryMessage>();
+
+        let future = sub.send(RouteQueryMessage::data_indefinite_route_request(
+            Some("0.0.0.0".to_string()),
```

Comment:
I'd like to see a more complex string addr like `somethingbefore0.0.0.0somethingafter`.